pkg/didcomm/crypto/jwe/authcrypt/authcrypt.go

```diff
@@ -26,9 +25,6 @@ const C20P = ContentEncryption("C20P") // Chacha20 encryption + Poly1035 authent
 // XC20P XChacha20Poly1035 algorithm
 const XC20P = ContentEncryption("XC20P") // XChacha20 encryption + Poly1035 authenticator cipher (192 bits nonce)
 
-// randReader is a cryptographically secure random number generator.
-var randReader = rand.Reader
```

Comment:
although not currently manipulated in tests, 
but this variable could be used to force random repeatable values in tests (inspired from go-jose).

Reply:
These things should be documented / commented. Global variables raises flags.

Reply:
Also - we shouldn't do these things until we actually are using them :).

Reply:
sure.. it can be removed then